docs/src/pages/guides/pickers-migration/pickers-migration.md

```diff
@@ -0,0 +1,118 @@
+# Upgrading from @material-ui-pickers
+
+<p class="description"> @material-ui/pickers was moved to the @material-ui/lab </p>
+
+## ☢️ Attention
+
+Pickers were entirely changed. In most places, the logic was rewritten from scratch, so it is not possible to maintain the whole list of changes. Here is an overview of the most important concepts that were changed. If you are going to migrate – the easiest way will be to go through the component usages in your codebase and rewrite them one-by-one. Do not forget to run your tests after all!
+
+This guide is an overview of the core concepts that were changed from the pickers v3.2.10.
+
+## ⬇️ Install
+
+First of all you need to install the `@material-ui/lab` if it's not installed. Nothing more required.
+
+## 🌚 Imports
+
+We do not expose the `keyboard` version of pickers anymore. All the version of mobile and desktop pickers implements keyboard inputs for accessibility reasons.
+
+```diff
+- import { KeyboardDatePicker } from '@material-ui/pickers'
++ import { DatePicker } from '@material-ui/lab'
+
+- <KeyboardDatePicker />
++ <DatePicker />
+```
+
+Also instead of `variant` prop specific wrappers of components were moved to different imports. So you will not bundle the `Dialog` if you are using only desktop picker view.
```

Comment:
```suggestion
Also, instead of providing a `variant` prop, these were moved to different imports, meaning that your bundle won't  include `Dialog` if you are using only the desktop picker.
```